src/formatDate.js

```diff
@@ -50,7 +50,46 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFromFormat = fromFormat.pop();
+  const separatorToFormat = toFormat.pop();
+  const numbersOfDate = date.split(separatorFromFormat);
+  const currentFormat = {};
+  const correctFormat = [];
+  let i = 0;
+
+  for (const format of fromFormat) {
+    currentFormat[format] = numbersOfDate[i];
+    i++;
+  }
+
+  let yearsFormat = 0;
+
+  for (const newFormat of toFormat) {
+    if (!fromFormat.includes(newFormat) && newFormat === 'YY') {
+      yearsFormat = currentFormat['YYYY'] % 100;
+      correctFormat.push(yearsFormat);
+      continue;
```

Comment:
```suggestion
```
you can remove `continue`

Reply:
I can't remove continue because my program must go to next iteration if fromFormat includes 'YY' or 'YYY', because if this conditions aren't done, i push number in array with correct format.